packages/react-arborist/src/types/renderers.ts

```diff
@@ -9,6 +9,7 @@ export type NodeRendererProps<T> = {
   node: NodeApi<T>;
   tree: TreeApi<T>;
   dragHandle?: (el: HTMLDivElement | null) => void;
+  previewHandle?: (el: HTMLDivElement | null) => void; 
```

Comment:
How does this previewHandle get used?

Reply:
it will be used in nodeRender, if user need. They can custom preview div ref.
such as :

<div ref={previewHandle}><dragImg ref={dragHandle} />im a test</div>

Reply:
Oh ok, I see.